app/services/site_channel_verifier.rb

```diff
@@ -107,30 +107,25 @@ def verify_site_channel_dns
 
   def verify_site_channel_public_file
     generator = SiteChannelVerificationFileGenerator.new(site_channel: channel)
-    uri = URI("https://#{channel.details.brave_publisher_id}/.well-known/#{generator.filename}")
-    response = fetch(uri: uri)
-    if response.code == "200"
-      token_match = /#{channel.details.verification_token}/.match(response.body)
-      if token_match
+
+    host_inspector = SiteChannelHostInspector.new(
+      brave_publisher_id: generator.url,
```

Comment:
`brave_publisher_id` is used to refer to the root domain of the site.  It looks like `generator.url` includes the path to the public file. If the we will be using the SiteChannelHostInspector more generally, can we change the name of this parameter?

Reply:
Renamed it to be `url`

https://github.com/brave-intl/publishers/pull/1410/commits/8720d181801543d2ad5240b6e54fcc1d9371f31c